### ISSUE TITLE
PhoneNemberPicker does not display contact list

### ISSUE THREAD

**Issue by GoogleCodeExporter** [niteshmourya/app-inventor-for-android#653]
```
This form is only for reporting bugs in App Inventor.   Do not use it for
reporting problems with connecting to the phone or opening the Blocks
Editor.  Those matters typically involve needing the right drivers, being
blocked by firewalls, or interference from other software running on  your
computer.  Ask about these in the group Forum:
http://groups.google.com/group/appinventor

Also use the Forum -- not this form -- to request  help in writing apps,
and other discussions,  please visit:

For bugs:
Use this space to describe the issue in detail. Be sure to include:
- PlacePhoneNumberPicker in viewer, click on button on phone
- What happened: The phone shows a empty window with a search for contacts box 
at the top, a watermark in the background says no contacts 
(I have lots)
- What you think the correct behavior should be. Should open my contacts list

Attach any material you think would be relevant to diagnosing the problem,
such as:
-  The source of your app: use "download to computer" to get the source zip
file: I dont neeed to because I was just following the turtorial "PicCall"
-  A screen shot: None

Don't forget to mention which device you have, and which version of Android
is installed on it.  (Find it under Home > Menu > Settings > About
phone.)Please also tell us what Operating System your computer is running
and what browser you were using to access App Inventor.

My phone is:
Samsung Captivate (galaxy S)
Running cognition v2.2.Beta9 ROM (leaked Froyo based on i897ucji6)
```

Original issue reported on code.google.com by `leadclim...@gmail.com` on 7 Nov 2010 at 3:43
- Merged into: #202
